data/fields/barrier.json

```
@@ -1,5 +1,32 @@
 {
     "key": "barrier",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
+         "options": {
+             "fence": "Fence",
+             "gate": "Gate",
+             "wall": "Wall",
+             "hedge": "Hedge",
+             "kerb": "Kerb",
```

```suggestion
             "kerb": "Curb",
```

_Kerb_ is British English, if I'm not mistaken.